### PR TITLE
docs: multi repo guide (wip)

### PR DESCRIPTION
**Pull request by eysi09**
The very beginning of a guide to the multi repo functionality.

### CODE REVIEW

File: docs/guides/remote-sources.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Importing remote sources
+Garden can import arbitrary remote sources into a project. A remote source can contain any number of modules and is treated just like any other directory within the project. It's simply a directory that contains one or many Garden modules and is hosted in a remote repository.
+
+To import remote sources we add their `name` and `respositoryUrl` under the `sources` key in the project level garden.yml config file:
+```yml
+project:
+  name: my-project
+  sources:
+    - name: source-name # name of remote source
+      repositoryUrl: https://my-git-server.com/source-name.git # git repository URL (can also point to a local file)
```
Comment (edvald) [inline docs/guides/remote-sources.md]: "file" -> "directory" ?

File: docs/guides/remote-sources.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+### Example
+
+For example, let's say the our project has the following structure:
```
Comment (edvald) [inline docs/guides/remote-sources.md]: "For example, let's" -> "Let's"

Comment (edvald) [reply docs/guides/remote-sources.md]: Oh and -_the_

File: docs/guides/remote-sources.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Working with remote sources
+
+With the Garden framework you can import remote sources and modules into your project, and run them just like your local modules and services. This is useful if your project's source code is spread across multiple repositories.
```
[the remark this answers is not in the document]
Comment (eysi09) [reply docs/guides/remote-sources.md]: Agreed.

**Comment by ellenkorbes** [comment]
I've moved these to the `allthedocs` branch with all the suggestions. I think we can close this.

**Pull request closed without merging**